CONTRIBUTING.md

```diff
@@ -1,123 +1,55 @@
 # CONTRIBUTING GUIDELINES
-
-Oh-My-Bash is a community-driven project. Contribution is welcome, encouraged and appreciated.
-It is also essential for the development of the project.
+Oh-My-Bash is Free and Open Source project under the terms of GNU General Public License v2.0 relying on contributions from third parties.
 
 These guidelines are an attempt at better addressing the huge amount of pending
 issues and pull requests. Please read them closely.
 
-Foremost, be so kind as to [search](#use-the-search-luke). This ensures any contribution
-you would make is not already covered.
-
-* [Issues](#reporting-issues)
-  * [You have a problem](#you-have-a-problem)
-  * [You have a suggestion](#you-have-a-suggestion)
-* [Pull Requests](#submitting-pull-requests)
-  * [Getting started](#getting-started)
-  * [You have a solution](#you-have-a-solution)
-  * [You have an addition](#you-have-an-addition)
-* [Information sources (_aka_ search)](#use-the-search-luke)
-
-**BONUS:** [Volunteering](#you-have-spare-time-to-volunteer)
-
-## Reporting Issues
-
-### You have a problem
-
-Please be so kind as to [search](#use-the-search-luke) for any open issue already covering
-your problem.
-
-If you find one, comment on it so we can know there are more people experiencing it.
-
-If not, look at the [Troubleshooting](https://github.com/ohmybash/oh-my-bash/wiki/Troubleshooting)
-page for instructions on how to gather data to better debug your problem.
-
-Then, you can go ahead and create an issue with as much detail as you can provide.
-It should include the data gathered as indicated above, along with:
-
-1. How to reproduce the problem
-2. What the correct behavior should be
-3. What the actual behavior is
-
-Please copy to anyone relevant (_eg_ plugin maintainers) by mentioning their GitHub handle
-(starting with `@`) in your message.
-
-We will do our very best to help you.
-
-### You have a suggestion
-
-Please be so kind as to [search](#use-the-search-luke) for any open issue already covering
-your suggestion.
-
-If you find one, comment on it so we can know there are more people supporting it.
-
-If not, you can go ahead and create an issue. Please copy to anyone relevant (_eg_ plugin
-maintainers) by mentioning their GitHub handle (starting with `@`) in your message.
-
-## Submitting Pull Requests
-
-### Getting started
-
+## Merge Requests
```

Comment:
As this is GitHub (but not GitLab or other services), it is more natural to call it Pull Requests. Any thoughts?